### PR TITLE
Add PubMatic bidder doc file and in-app video to bidder info

### PR DESCRIPTION
**Pull request by mike-chowla**
Add PubMatic bidder doc file
Add app video capability to PubMatic bidder info file

### CODE REVIEW

File: docs/bidders/pubmatic.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# PubMatic Bidder
+
+## Test Request
+
+The following test parameters can be used to verify that Prebid Server is working properly with the 
+PubMatic adapter. This example includes an `imp` object with an PubMatic test publisher ID, ad slot,
+and sizes that would match with the test creative.
+
```
Comment (guscarreon) [inline docs/bidders/pubmatic.md:8]: Are we ok with only mentioning this example and two out of the four parameters that the `imp.ext` object can come with? What about a video `imp` example? Will that be left for another PR?

**Comment by SyntaxNode** [comment]
@mike-chowla Could you please take a look at @guscarreon's comment?

**Comment by mike-chowla** [comment]
@SyntaxNode Like I said above, the PR submitted follows the conventions of the existing doc files.  If the Prebid Server PMC wants to add a rule that all bidder doc files must document every accepted parameter, that's fine but that rule should be written down somewhere and efforts made to update all existing bid doc files to conform to the rule.

**Comment by SyntaxNode** [comment]
@mike-chowla There is no response from you above, which is why I asked for you to look at @guscarreon's comment. I'll infer from this response that you don't see a need to for additional examples. That's fine. It was just a question.

**Comment by guscarreon** [comment]
> @mike-chowla There is no response from you above, which is why I asked for you to look at @guscarreon's comment. I'll infer from this response that you don't see a need to for additional examples. That's fine. It was just a question.

I'm good with the changes, sorry I didn't come back sooner

**Comment by mike-chowla** [comment]
@guscarreon no worries.  I had put a response on code review comment but looks like I failed to submit it.  Apologies for the confusion
